content/2-3.md

```diff
@@ -1,44 +1,46 @@
 ---
-title: "[Draft] Module 3: Images and Graphics"
-permalink: /curricula/developing-accessible-content/images-and-graphics/
-ref: /curricula/developing-accessible-content/images-and-graphics/
+title: "[Draft] Module 3: Images"
+permalink: /curricula/developer-modules/images/
+ref: /curricula/developer-modules/images/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-3.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/navigational-menus/
-  next: /curricula/developing-accessible-content/tabular-information/
+  previous: /curricula/developer-modules/menus/
+  next: /curricula/developer-modules/tables/
 ---
 
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
 
-* explain use of proper coding to make images and graphics accessible
-* describe types of images from an accessibility perspective
 * demonstrate how people with disabilities use text alternatives to access information contained in images and graphics
+* explain coding techniques to provide text alternatives for images and graphics
```

Comment:
What is the difference between an "image" and a "graphic"? If there is a substantive difference should they be presented separately? If not, can they all just be referenced as "images"?

Reply:
I will take a general pass on this module and clean these up. Especially now that we are proposing to call this module just "Images", I think it makes sense.